main.go

```diff
@@ -44,7 +44,7 @@ func RunCustom(args []string, commands map[string]cli.CommandFactory) int {
 	// users should not be running should be placed here, versus hiding
 	// subcommands from the main help, which should be filtered out of the
 	// commands above.
-	hidden := []string{"check", "executor", "syslog"}
+	hidden := []string{"check", "executor", "syslog", "apply"}
```

Comment:
Put in in the above commandsInclude list.

`acl policy, acl policy apply, acl token, acl token create`